### ISSUE TITLE
Deinonychus engine generates insane amounts of waste heat

### ISSUE THREAD

**Issue by serp777** [sswelm/KSP-Interstellar-Extended#697]
Using the Deinonychus engine in campaign and it generates absolutely ridiculous amounts of waste heat once the thrust has been upgraded. Also the thrust doesn't actually increase. THe waste heat is so rapid that it almost instantly causes nuclear engines with a built in heat sink to just explode as the heatsink gets to max temp. Its like 100 times more waste heat than a regular engine. I can see it generating gigawatts of waste heat so only the largest heat sinks are sufficient for cooling this very low thrust engine.

**Comment by Cholerix** [comment]
Internally this engine is treated as a [vista fusion engine](https://github.com/sswelm/KSP-Interstellar-Extended/blob/3bd0e62a97b273d19b4f2db06804f2d40899eed7/FNPlugin/Propulsion/VistaECU2.cs#L9). I don't know why or how. Since the waste heat of the Vista depends on the upgrade level, the methane engine starts generating insane amounts waste heat once upgraded.

**Comment by sswelm** [comment]
Yes that is not intended. I believe the reason I used this engine was because of it ability to upgrade. THe waste heat is caused by some default behavior

**Comment by serp777** [comment]
What's weird is I compared the cfg files between the rutherford and the methane engine, and they seem to be identical. Waste heat multiplier is set to 1 even for all upgrades; fusion heat is set to 0 as well. I don't see any obvious reason why the upgrade would trigger significant waste heat generation. As far as I can tell the only thing that's changed is the thrust when you do the upgrade.
